core/profiles/standard/standard.install

```diff
@@ -63,7 +63,7 @@ function standard_install() {
   $filtered_html_format = (object) $filtered_html_format;
   filter_format_save($filtered_html_format);
 
-  // Changed display name from Full HTML to Raw HTML but keeping machine name 
+  // Changed display name from Full HTML to Raw HTML but keeping machine name
```

Comment:
Tense here should be present: `// Change display...`

Reply:
Oh, just seeing now that this line only had a whitespace change. Maybe ignore this.